.ci/compatible_gem_version

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+require 'net/http'
+require 'json'
+
+GEM_TO_TEST           = "rubygems-update"
+
+if ARGV.count > 0
+  RUBY_VERSION_TO_MATCH = ARGV[0]
+else
+  RUBY_VERSION_TO_MATCH = RUBY_VERSION
+end
+
+API_URL = "https://rubygems.org/api/v1/versions/#{GEM_TO_TEST}.json"
+
+# Load list of all available versions of GEM_TO_TEST
+gem_versions = JSON.parse(Net::HTTP.get(URI(API_URL)))
+
+# Process list to find matching Ruby version
+matching_gem = gem_versions.find { |gem|
+  Gem::Dependency.new('', gem['ruby_version']).
+    match?('', RUBY_VERSION_TO_MATCH)
+}
+
+puts matching_gem['number']
+#puts "Latest version of #{GEM_TO_TEST} " +
+#     "compatible with Ruby #{RUBY_VERSION_TO_MATCH} " +
+#     "is #{matching_gem['number']}."
```

Comment:
Did you intend to leave this here?

Reply:
it was practical to test. I can remove or hide behind a DEBUG.